backend/shared/src/main/scala/zio/logging/package.scala

```diff
@@ -0,0 +1,22 @@
+package zio
+
+package object logging {
```

Comment:
Remember to PR to ZIO!

Reply:
I will merge this first and then open PRs for annotation and `Cause` in `Zlogger`